### PR TITLE
Fix spelling in saltstack layer

### PR DESCRIPTION
**Pull request by Johnstone-Tech**

### CODE REVIEW

File: layers/+tools/salt/README.org [hunk 1]
```diff
@@ -12,7 +12,7 @@ This layer provides syntax highlighting for Saltstack files.
```
Comment (d12frosted) [inline layers/+tools/salt/README.org]: > This layer provides syntax highlighting for Saltstack files.

Hey, could you fix the naming here as well?

Comment (Johnstone-Tech) [inline layers/+tools/salt/README.org]: I think it may be best to keep it as saltstack here as the product is known as saltstack to many.

Comment (d12frosted) [inline layers/+tools/salt/README.org]: Oh. I thought this is the line you changed in previous PR. My bad.

**Comment by d12frosted** [comment]
Thank you :+1: Cherry-picked into `develop`, you can safely delete your branch.

**Pull request closed without merging**